docs/telemetry.md

```diff
@@ -0,0 +1,82 @@
+# Data Layer Observer Telemetry
+
+DLO's telemetry implementation is inspired by [Open Telemetry](https://opentelemetry.io/). While DLO doesn't depend on Open Telemetry, DLO telemetry can be configured to be measured by another provider like Open Telemetry or exported to a custom destination for visibility.
+
```

Comment:
Might be a good idea to quickly mention that DLO can easily be configured to log to the console be default, with a link back to the `## Telemetry` header in the Readme. Good to cross-reference.